pkg/tsdb/influxdb/flux/builder.go

```diff
@@ -119,40 +119,42 @@ func (fb *frameBuilder) Init(metadata *query.FluxTableMetadata) error {
 		}
 	}
 
+	// Timeseries has a "_value" and a time
 	if fb.isTimeSeries {
 		col := getTimeSeriesTimeColumn(columns)
-		if col == nil {
-			return fmt.Errorf("no time column in timeSeries")
+		if col != nil {
+			fb.timeColumn = col.Name()
+			fb.timeDisplay = "Time"
+			if "_time" != fb.timeColumn {
+				fb.timeDisplay = col.Name()
+			}
+			return nil
 		}
+	}
 
-		fb.timeColumn = col.Name()
-		fb.timeDisplay = "Time"
-		if "_time" != fb.timeColumn {
-			fb.timeDisplay = col.Name()
+	// reset any timeseries properties
+	fb.value = nil
+	fb.isTimeSeries = false
+	fb.labels = make([]string, 0)
+	for _, col := range columns {
+		// Skip the result column
```

Comment:
Why skip the result column? What data is in it?

Reply:
That is from:
https://github.com/grafana/grafana/pull/27081

when things are not a timeseries... they are just empty

Reply:
see also https://github.com/grafana/grafana/issues/26927

Reply:
(note the only change here is just indentation